src/html/index.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>Test</title>
+    <title>Portal</title>
+    <link rel="icon" href="/src/assets/favicon.ico" type="image/x-icon" />
```

Comment:
src folder is not available in production release. Edit compile and dev scripts to include favicon.ico file in build directory.